tribe-ext-community-currency-symbol/index.php

```diff
@@ -0,0 +1,408 @@
+
+		// Setup fields on the settings page
+		$setting_helper = new Tribe__Extension__Settings_Helper();
+
+		$fields = array();
+
+		// List of allowed symbols
+		$fields['tribe_ext_ce_cost_currency_symbols'] = array(
+			'type'            => 'text',
+			'label'           => esc_html__( 'Event Cost: Allowed Currency Symbols', 'tribe-extension' ),
+			'tooltip'         => esc_html__( 'Pipe-separated list of allowed currency symbols. (Once this option is set and saved, you will be able to pick the Default Currency Symbol for new events.) The drop-down will display these in the order you enter them. Example 1: $|€|£|Fr|¥|CNY|C$|AU$. Example 2: USD|EUR|GBP|JPY|CNY|CAD|AUD', 'tribe-extension' ),
+			'validation_type' => 'textarea',
+		);
+
+		// only allow modifying after the List exists (to be able to pick from)
+		if ( tribe_get_option( 'tribe_ext_ce_cost_currency_symbols' ) ) {
+			$fields['tribe_ext_ce_cost_currency_symbol_default'] = array(
+				'type'            => 'dropdown',
+				'label'           => esc_html__( 'Event Cost: Default Currency Symbol', 'tribe-extension' ),
+				'tooltip'         => esc_html__( 'The default currency symbol for new events (does not apply when editing existing events). Leave blank to not set a default.', 'tribe-extension' ),
+				'options'         => $this->symbol_list_array( true ),
+				'validation_type' => 'options',
+			);
+		}
+
+		// List of allowed symbols
+		$fields['tribe_ext_ce_cost_currency_symbol_position_disabled'] = array(
+			'type'            => 'checkbox_bool',
+			'label'           => esc_html__( 'Event Cost: Disallow Currency Position Selector', 'tribe-extension' ),
+			'tooltip'         => esc_html__( 'If checked, event creators will not be able to change the currency position (before or after the cost) and your site default from the General settings tab will be used.', 'tribe-extension' ),
+			'default'         => false,
+			'validation_type' => 'boolean',
+		);
+
+		$setting_helper->add_fields(
+			$fields,
+			'community',
+			'single_geography_mode' // At the bottom of the "Form Defaults" section
+		);
+	}
+
+	/**
+	 * Get list of allowed symbols as an array.
+	 *
+	 * @param bool $prepend_empty
+	 *
+	 * @return array
+	 */
+	public function symbol_list_array( $prepend_empty = false ) {
+		$setting = tribe_get_option( 'tribe_ext_ce_cost_currency_symbols' );
+
+		if ( '' === (string) $setting ) {
+			return array();
+		}
+
+		$settings = explode( '|', $setting );
+
+		/**
+		 * Filters the array of currency symbols passed to the drop-down.
+		 *
+		 * @param array $settings The original array.
+		 */
+		$settings = (array) apply_filters( 'tribe_ext_ce_cost_currency_symbol_list_array', $settings );
+
+		$symbols = array();
+
+		foreach ( $settings as $key => $value ) {
+			$value = esc_attr( sanitize_text_field( $value ) );
+			if ( '' !== $value ) {
+				$symbols[ $value ] = $value;
+			}
+		}
+
+		if ( ! empty( (bool) $prepend_empty ) ) {
```

Comment:
Would this fail on PHP 5.2? It's only in 5.3+ that you can use `empty()` on expressions rather  than variables. 

It probably makes the most sense to just strictly check this. We know it will be declared, so the `===` operator might be well suited for this task. `if ( false === $prepend_empty ) {`